target_chains/near/receiver/workspace-test.sh

```diff
@@ -17,4 +18,4 @@ cp target/wasm32-unknown-unknown/release/pyth.wasm .
     cp target/wasm32-unknown-unknown/release/wormhole_stub.wasm ../receiver
 )
 
-RUST_LOG=info cargo nextest run
+RUST_LOG=info cargo test
```

Comment:
I guess we can remove nextest from `.github/workflows/ci-near-contract.yml` as well.